api/src/main/resources/script.sql

```diff
@@ -3,20 +3,24 @@ CREATE TABLE `contacts` (
     id VARCHAR(36)  UNIQUE,
     first_name VARCHAR(100) NOT NULL,
     last_name VARCHAR(100) NOT NULL,
-    name VARCHAR(155) NOT NULL,
+    phone VARCHAR(155) NOT NULL,
 
     PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
-CREATE TABLE `users` (
+CREATE TABLE `teams` (
     email VARCHAR(155) UNIQUE,
     password VARCHAR(500) DEFAULT NULL,
-    first_name VARCHAR(45) DEFAULT NULL,
-    last_name VARCHAR(45) DEFAULT NULL,
+    name VARCHAR(45) DEFAULT NULL,
```

Comment:
This is team name so we don't need first and last.

Reply:
Could you also update the hibernate object and the dtos associated with it?ew